arch/amd64-options.mk

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Intel x86-64 settings
+
+MACHINETYPE := pc
```

Comment:
not sure, but I think this should be inside an if sentence

```make
ifeq ($(ARCH),x86_64)
   MACHINETYPE := pc
endif
```

Reply:
I think we should be protected there from the check in the top-level `Makefile` ?